include/functions.h

```diff
@@ -358,12 +358,12 @@ void Actor_Kill(Actor* actor);
 void Actor_SetFocus(Actor* actor, f32 yOffset);
 void Actor_SetScale(Actor* actor, f32 scale);
 void Actor_SetObjectDependency(PlayState* play, Actor* actor);
-void func_8002D7EC(Actor* actor);
-void func_8002D868(Actor* actor);
-void Actor_MoveForward(Actor* actor);
-void func_8002D908(Actor* actor);
-void func_8002D97C(Actor* actor);
-void func_8002D9A4(Actor* actor, f32 arg1);
+void Actor_UpdatePos(Actor* actor);
+void Actor_UpdateVelocityWithGravity(Actor* actor);
+void Actor_MoveWithGravity(Actor* actor);
+void Actor_UpdateVelocityWithoutGravity(Actor* actor);
+void Actor_MoveWithoutGravity(Actor* actor);
+void func_8002D9A4(Actor* actor, f32 speed);
 s16 Actor_WorldYawTowardActor(Actor* actorA, Actor* actorB);
```

Comment:
Should `Actor_UpdatePosByAnimation` be in here? What's the future intent/style for declarations of completely unused functions, but seem to be designed just fine and could be used externally? Would that be something to wait for more split header files?

Reply:
any function that seems like it is meant to be called externally will be exposed. personally dont really care for functions.h that much though, cause it will just be fixed when proper headers exist. we were very inconsistent about what gets included in functions.h in the decomp phase, so its a problem everywhere